test/validators/movie.test.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieValidator = require('../../lib/validators/movie');
+
+describe('movie validator', () => {
+
+  it('succeeds when all allowed params are included', () => {
+    const title = 'title';
+    const release_year = 2017;
+    const payload = { title, release_year };
+    const result = Joi.validate(payload, MovieValidator);
+
+    expect(result.error).to.be.null;
+  });
+
+  describe('title', () => {
+
+    it('is required', () => {
+      const payload = {};
+      const result = Joi.validate(payload, MovieValidator);
+
+      expect(result.error.details[0].path).to.eql('title');
+      expect(result.error.details[0].type).to.eql('any.required');
+    });
+
+    it('is less than 255 characters', () => {
+      const title = 'a'.repeat(256);
+      const payload = { title };
```

Comment:
Something that we usually do (you don't have to do it here but it's good to have in mind) is having a default, valid payload defined in a `beforeEach` condition. For example:
```js
let payload;

beforeEach(() => {
  payload = {
    title: 'My Title',
    release_year: 2017
  };
});
```
This payload can then be modified in each test to make it fail however you may want to. This avoids defining a new payload in each test and you only need to modify the property that you need it to fail for.
```js
// If you need to remove the title
delete payload.title;

//If yo need a really long title 
payload.title = 'a'.repeat(256);
```